src/test/java/org/broadinstitute/hellbender/tools/walkers/sv/SVAnnotateEngineUnitTest.java

```diff
@@ -581,6 +744,54 @@ public Object[][] getAnnotateStructuralVariantTestData() {
                         createAttributesMap(
                                 Arrays.asList(GATKSVVCFConstants.PROMOTER, GATKSVVCFConstants.INTERGENIC),
                                 Arrays.asList("EMMA1", true)) },
+                // CPX with dDUP CG and promoter annotations
+                { createVariantContext("chr1", 10, 11, null, null, null,
```

Comment:
It'd be nice to have all the CPX types tested here: `dupINVdup`, `delINVdup`. Also I was wondering about a gene that fully lies within the DUP interval of a CPX event like `dupINVdup` -- I think it will work but it'd be nice to see an example tested here where that gets a copy gain annotation.

Reply:
Added dupINVdup with copy gain. There are still 5 complex types not represented in this particular unit test (although all types are represented in testGetSegmentsFromComplexIntervals, a few more are in testAnnotateComplexEvents, and I tried to cover all the interval modifications that are done for noncoding and TSS annotations)